manifests/init.pp

```diff
@@ -66,10 +83,19 @@
   class { '::shibboleth_idp::attribute_resolver': } ->
   class { '::shibboleth_idp::attribute_filter': }
 
+  class { '::shibboleth_idp::service':
+    service_name    => $service_name,
```

Comment:
Somehow I did this halfway. The params and service classes already had these values. I just had to fix up the class here.